### ISSUE TITLE
[Bug] underscore within math formula recognized as Italic mark

### ISSUE THREAD

**Issue by MForever78** [HappenApps/Quiver#689]
OS: 10.11.6
Version: 3.0.3 (88)
Description:

For example:

```
Move the ball $\Delta v_1$ towards direction $v_1$
```

The editor renders it as:

![image](https://cloud.githubusercontent.com/assets/4675950/18305356/ca3f3054-7519-11e6-9ec2-a5eb126573ae.png)

It can't be solved by adding escape character because that would change the semantic of the LaTeX symbol.

**Comment by MForever78** [comment]
@ylian Is this a "won't fix"? I assume this bug may relates to some third party code. If that is the case the issue should be closed and I'll move on to another editor.

**Comment by ylian** [comment]
Indeed it's a defect with the underlying Ace editor. Not really a "won't fix" since I will change the editor for the Markdown cell.

Also, to set the expectation right, Quiver is a note-taking app, not a dedicated Markdown editor. So you may be disappointed to use Quiver as such.

**Comment by MForever78** [comment]
Thanks for the reply. Glad to hear that this may be fixed.

As for the usage part, I'm using Quiver to take note on the class using Markdown (and LaTeX of course). I don't think it is against the definition of "note-taking app", is it?

Anyway, thanks for the good app and your effort. Hope this could be fixed soon.

**Comment by ylian** [comment]
Yes, that is certainly the use case Quiver is built for. But at this moment Quiver's Markdown editing function still needs much work.
